### ISSUE TITLE
version information in `rdctl list-settings` output

### ISSUE THREAD

**Issue by gunamata** [rancher-sandbox/rancher-desktop#3054]
It would be nice to include the application version information in the `rdctl list-settings` output. At present, there's a `version` attribute in the json output but this refers to the rdctl version I guess? It will be helpful to include the Rancher Desktop application version itself. For the CI builds, we should see whatever the version value we see in the `General` tab.

**Comment by jandubois** [comment]
I disagree that it should be part of the `settings` output (because it is not a setting), but we want something like `rdctl info` that shows the version and various other information about the installed version (like the location of various data directories).

**Comment by jandubois** [comment]
I would also think that the app version should be included in the output of

```console
$ rdctl version
rdctl client version: 1.1.0, targeting server version: v0
```

One challenge is to determine this information when the app itself is not running, without duplicating the configuration logic, which could then get out-of-sync.

**Comment by jandubois** [comment]
> The version field in `list-settings` is for the settings format itself.

It is indeed, but that isn't a (user-changeable) setting either, so also shouldn't be displayed.

Right now `rdctl list-settings` displays the same output as `rdctl api settings`. We need to change that, so the output is not in JSON format, but shows each setting with the option name that can be passed to `rdctl set` and `rdctl start`, e.g. something like

```console
$ rdctl list-settings
container-engine    moby
kubernetes-enabled  true
kubernetes-version  1.25.2
```
